content/en/docs/reference/config/analysis/ist0113/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: MTLSPolicyConflict
+layout: analysis-message
+---
+
+This message occurs when a destination rule resource and a policy resource are
+in conflict with regards to mutual TLS. The two resources are in conflict if they each
+specify incompatible mutual TLS modes to be used; this conflict means traffic matching
+the destination rule to the specified host will be rejected.
+
+This message is only produced on clusters that are not using [automatic mutual
```

Comment:
should this be "service meshes" instead of "clusters"?

Reply:
Done.